src/components/templates/Auditorium/components/AllSectionPreviews/AllSectionPreviews.tsx

```diff
@@ -99,12 +99,7 @@ export const AllSectionPreviews: React.FC<SectionPreviewsProps> = ({
 
   return (
     <>
-      {parentVenue && (
-        <BackButton
-          onClick={backToParentVenue}
-          locationName={parentVenue.name}
-        />
-      )}
+      {parentVenue && <BackButton onClick={backToParentVenue} />}
```

Comment:
Why was `locationName` removed here? We want to keep this on the main platform, it should only be removed on `env/github`.

Reply:
fixed in 088c807569a37b13ce953e79d623431827421b6a